Documentation/Registry.md

```diff
@@ -1379,7 +1391,7 @@ paths:
               schema:
                 type: integer
             Content-Version:
-              $ref: "#/components/headers/contentVersion"
+              $ref: "#/components/headers/optionalContentVersion"
```

Comment:
AFAIK we can't override `$ref`. 

Ideally we'd want to do something like this:

```suggestion
              $ref: "#/components/headers/contentVersion"
              required: false
```

Reply:
cc @czechboy0

Reply:
That's right, whether a header is required is part of the content, so to have an optional variant of the same header, we have to fully duplicate it.